src/js/vault-client/vault-api.js

```diff
@@ -7,6 +7,7 @@ import conf from '../configuration';
 
 const VAULT_BASE_URI = conf.VAULT.VAULT_BASE_URI;
 const VAULT_KV_ENGINE = conf.VAULT.VAULT_KV_ENGINE;
+const NB_DAYS_BEFORE_PWD_RENEWAL = 7;
```

Comment:
I suggest we move this to an environment variable. People may want to disable / tune this behaviour on their installation